sn_networking/src/lib.rs

```diff
@@ -72,7 +72,7 @@ use tracing::warn;
 pub(crate) const CLOSE_GROUP_SIZE: usize = 8;
 
 // Timeout for requests sent/received through the request_response behaviour.
-const REQUEST_TIMEOUT: Duration = Duration::from_secs(10);
+const REQUEST_TIMEOUT: Duration = Duration::from_secs(30);
```

Comment:
Perhaps a rename to `REQUEST_TIMEOUT_DEFAULT` to highlight this is a fallback value?

I do prefer defaults to be more higher up though, so it could also be an idea to not make `new_client` accept an `Option` and specify this default somewhere in `main`.

Reply:
What is the reasoning about increasing it from `10` to `30`?

Reply:
Folk were complaining things were timing out, so I thought default maybs should go up.